src/Core/Api/APIService.php

```diff
@@ -91,7 +91,7 @@ public function query($query, $method = 'post', $variables = []) {
     $this->loggerContract
         ->debug(TranslationHelper::getLoggerKey('apiService'), ['additionalInfo' => [
           'url' => $url,
-          'arguments' => $arguments
+          'arguments' => [$url, $arguments[1]['json']]
```

Comment:
Rather than remove a header by index (which happens to be `Authentication` due to the code above it), please look into explicitly replacing the value of this specific header (but preserving one or two characters so we can compare it to what we think it should be) instead. 

Removing the header will make us think we are not setting the header at all.